apps/pie-docs/src/_11ty/shortcodes/usage.js

```diff
@@ -0,0 +1,88 @@
+const pieIconsSvg = require('../filters/pieIconsSvg');
+const pieDesignTokenColours = require('../filters/pieDesignTokenColours');
+const list = require('./list');
+const usageTypes = require('../../_data/usageTypes');
+
+const metadata = {
+    do: {
+        displayName: 'Do',
+        styleColour: 'support-positive',
+        iconFill: 'support-positive',
+        iconName: 'check-circle',
+    },
+    dont: {
+        displayName: "Don't",
+        styleColour: 'support-error',
+        iconFill: 'support-error',
+        iconName: 'close-circle',
+    },
+};
+
+const buildImage = ({
+    width, alt, mobileSrc, src,
+}) => {
+    const isImageFullContainerWidth = !width;
+    const imageStyles = !isImageFullContainerWidth ? `style="--img-width: ${width};"` : ''; // If image isn't full width, set it to required width
+    const imageAlt = `alt="${alt || ''}"`;
+    const mobileImageMaxWidth = '600px';
+
+    return `<figure class="c-usage-img">
+          <picture>
+            ${mobileSrc ? `<source ${imageStyles} media="(max-width: ${mobileImageMaxWidth})" srcset="${mobileSrc}">` : ''}
+            <img src="${src}" ${imageStyles} ${imageAlt}>
+          </picture>
+      </figure>`;
+};
+
+const buildUsageCard = (usageType, { type, items }) => {
+    const {
+        iconName, iconFill, styleColour, displayName,
+    } = metadata[usageType];
+    const isImage = type === usageTypes.image;
+    const styleColourValue = pieDesignTokenColours({ tokenName: styleColour, tokenPath: ['alias', 'default'] });
+    const svg = pieIconsSvg({
+        name: iconName,
+        attrs: {
+            height: 24,
+            width: 24,
+            fill: iconFill,
+            'aria-hidden': 'true',
+        },
+    });
+    const backdropClasses = ['c-usage-backdrop', ...(isImage ? ['c-usage-backdrop--hasImage'] : [])];
+
+    const content = isImage
+        ? items.map((i) => buildImage(i)).join(' ')
+        : list({ type: 'bullet', items });
+
+    return `
+    <article class="c-usage" style="--style-colour: ${styleColourValue};">
+      <div class="c-usage-heading">
```

Comment:
I think you could update this to just use figure & figcaption. I had a play with the markup on the deploy, and something like this looks like it should work I think:

![Screenshot 2023-08-01 at 13 41 31](https://github.com/justeattakeaway/pie/assets/805184/2f3ff1f9-b12c-4940-a5bc-490a47a7c5a1)

Reply:
Figure elements accept any element that is considered 'flow content' ([there's a list of these here](https://developer.mozilla.org/en-US/docs/Web/HTML/Content_categories#flow_content)). So `ul` is actually fine inside a figure.

Reply:
updated